### PR TITLE
Configurable Volume Creation

### PR DESCRIPTION
**Pull request by ghost**
Make volume preparation deployment more flexible in terms of file system type,
devices and mountpoints.

### CODE REVIEW

File: src/deploy-prepare-volume.yaml [hunk 1]
```diff
@@ -21,19 +21,56 @@ parameters:
...
     default: ''
 
+  device:
+    type: string
+    description: "The name of the block device"
+    default: "vdb"
+
+  mountpoint:
+    type: string
+    description: "The directory to mount the filesystem to."
```
Comment (neki06) [inline src/deploy-prepare-volume.yaml:31]: No need for a dot here

**Review by neki06 — CHANGES_REQUESTED** [review]

File: src/deploy-prepare-volume.yaml [hunk 2]
```diff
@@ -21,19 +21,56 @@ parameters:
...
+    default: "/mnt"
+
+  filesystem_type:
+    type: string
+    description: "The name of the filesystem to create on the disk"
+    default: "ext4"
+
+  filesystem_label:
+    type: string
+    description: "The label to set on the new filesystem""
```
Comment (neki06) [inline src/deploy-prepare-volume.yaml]: Delete one double-quote at the end

**Review by neki06 — CHANGES_REQUESTED** [review]

**Comment by felixfiskare** [comment]
Looks fine

**Review by felixfiskare — CHANGES_REQUESTED** [review]
Next to neki06's comments everything looks fine

**Pull request closed without merging**